.gitignore

```diff
@@ -4,5 +4,11 @@ infra-*/terraform.tfstate
 infra-*/terraform.tfstate*
 infra-*/.terraform*
 infra-*/secrets.auto.tfvars
+*kubeconfig
+*terraform.tfstate*
+*terraform.lock.*
+.terraform
+*secrets.auto.tfvars
 my-notes
+my-notes.txt
```

Comment:
Why do we need these notes file in the first place?

Reply:
Ignored. Out of scope for this PR. This didn't come from me.